$OpenBSD: patch-package_conf_in,v 1.2 2004/05/07 01:37:50 dons Exp $

let hs-crypto work with the ghc package system, and with the regress
suite

--- package.conf.in.orig	2004-05-02 15:45:10.000000000 +1000
+++ package.conf.in	2004-05-02 15:46:41.000000000 +1000
@@ -0,0 +1,23 @@
+Package
+    {name = "crypto",
+#ifndef INPLACE
+     import_dirs = ["${PREFIX}/lib/hs-crypto/imports/HScrypto"],
+#else
+     import_dirs = ["${WRKDIST}"],
+#endif
+     source_dirs = [],
+#ifndef INPLACE
+     library_dirs = ["${PREFIX}/lib/hs-crypto"],
+#else
+     library_dirs = ["${WRKDIST}"],
+#endif
+     hs_libraries = ["HScrypto"],
+     extra_libraries = [],
+     include_dirs = [],
+     c_includes = [],
+     package_deps = ["haskell98","text"],
+     extra_ghc_opts = [],
+     extra_cc_opts = [],
+     extra_ld_opts = [],
+     framework_dirs = [],
+     extra_frameworks = []}
